locale/ar.po

```diff
@@ -0,0 +1,183 @@
+msgstr "هل لدى عائلة المريضة تاريخ من الاصابة في سرطان الثدي؟"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_cancer_history_person:"
+msgid "Does the woman have a personal history of breast cancer?"
+msgstr "هل لدى المريضة اصابة سابقة من سرطان الثدي؟"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_lump:"
+msgid "New lump in breast or axilla (i.e. armpit)"
+msgstr "كتلة في الثدي أو الإبط"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_nipple_changes:"
+msgid "Nipple changes (e.g. redness, flaky skin, inversion, discharge)"
+msgstr "تغيرات بالحلمة (احمرار، تغير في الشكل، تشققات... الخ)"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_operational_intervention:"
+msgid "Was there any operational intervention in the breast?"
+msgstr "هل اجرت المريضة اية عمليات جراحية في الثدي؟"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_operational_intervention_comment:"
+msgid "Other Surgery Type"
+msgstr "أخرى"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_operational_intervention_type:"
+msgid "Type of Breast Surgery"
+msgstr "نوع العلاج"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_skin_irritation:"
+msgid "Irritation or dimpling of breast skin"
+msgstr "تهيج أو تنقير في جلد الثدي"
+
+msgctxt "field:gnuhealth.patient.evaluation,breast_swelling:"
+msgid "Thickening or swelling of part of the breast"
+msgstr "سماكة أو تورم جزء من الثدي"
+
+msgctxt "field:gnuhealth.patient.evaluation,cancer_history_person:"
+msgid "Did the woman have any other cancer in the past?"
+msgstr "هل سبق للمريضة الاصابة في ايّ نوع آخر من السرطان؟"
+
+msgctxt "field:gnuhealth.patient.evaluation,diagnosis_date:"
+msgid "Date of diagnosis"
+msgstr "تاريخ التشخيص"
+
+msgctxt "field:gnuhealth.patient.evaluation,genetic_risks:"
+msgid "Genetic Information"
+msgstr "المعلومات الوراثية"
+
+msgctxt "field:gnuhealth.patient.evaluation,known_brca_mutation:"
+msgid "Known BRCA1/2 mutation carrier?"
+msgstr "هل أنت حاملة لجينات الإصابة بسرطان الثدي BRCA1 / 2؟"
+
+msgctxt "field:gnuhealth.patient.evaluation,sequence_number:"
+msgid "Sequence Number"
+msgstr "تصنيف الورم"
+
+msgctxt "field:gnuhealth.patient.evaluation,surgery_date:"
+msgid "Date of surgery"
+msgstr "تاريخ الجراحة"
+
+msgctxt "field:gnuhealth.patient.evaluation,woman_lactating:"
+msgid "Is the woman lactating / breast feeding?"
+msgstr "هل المريضة في حالة ارضاع؟"
+
+msgctxt "field:gnuhealth.patient.evaluation,woman_pregnant:"
+msgid "Is the woman pregnant?"
+msgstr "هل المريضة حامل؟"
+
+msgctxt "model:ir.message,text:evaluation_location_selection_left_breast"
+msgid "Left Breast"
+msgstr "الثدي الأيسر"
+
+msgctxt "model:ir.message,text:evaluation_location_selection_right_breast"
+msgid "Right Breast"
+msgstr " الثدي الأيمن"
+
+msgctxt "model:ir.message,text:evaluation_location_selection_both_breasts"
+msgid "Both Breasts"
+msgstr "كلا الثديين"
+
+msgctxt "selection:gnuhealth.patient.evaluation,breast_operational_intervention_type:"
+msgid ""
```

Comment:
Can this be removed with empty `msgid` and empty `msgstr`?

Reply:
yeah, this can be removed :+1: